client/UI.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+class UI extends React.Component{
+
+  render(){
+    return(
+      <div>
+        <div className={'navbar'}>
+          <img src="./images/home.svg" alt="home button"/>
+        </div>
+        <div className={'outerContainer'}>
+          <div className={'container card'}>
+            <img src={'./images/sun.svg'} alt='sun' width='200' height='200'></img>
+            <div className={'temp'}>Tempurature 72&deg;</div>
+            <div className={'location'}>Location: Los Angeles</div>
+            <div className={'buttonContainer'}>
```

Comment:
If your class names are going to be a string you don't need the `{ }` for example:

```JSX
<div className="some-class"></div>
```